metricbeat/module/golang/expvar/_meta/data.json

```diff
@@ -0,0 +1,60 @@
+{
+    "@timestamp": "2017-02-10T07:50:49.746Z",
+    "beat": {
+        "hostname": "MacBook-Pro.local",
+        "name": "MacBook-Pro.local",
+        "version": "6.0.0-alpha1"
+    },
+    "golang": {
+        "expvar": {
+            "cmdline": "./metricbeat -httpprof=127.0.0.1:6060 -e -v",
+            "fetches": {
+                "golang-expvar": {
+                    "events": 1,
+                    "failures": 0,
+                    "success": 1
+                },
+                "golang-heap": {
+                    "events": 1,
+                    "failures": 0,
+                    "success": 1
+                }
+            },
+            "libbeat.config.module.running": 0,
```

Comment:
It is supported in 2.4, but only if enabled. But I agree we should not worry about this currently for dynamic metricsets as it will add quite a bit complexity. You would have to scan the full json doc.